update search code sample in SDK (#284)
kuzzleio/documentation: src/sdk-reference/js/6/base-controller/introduction/index.md

---
layout: sdk.html.hbs
title: Introduction
description: BaseController class
order: 0
---

# BaseController

Base class for a [custom SDK controller]({{ site_base_path }}sdk-reference/js/6/extend-sdk/#define-a-custom-sdk-controller).  

> Custom SDK controllers must extends the `BaseController` class to by added to the SDK with the [Kuzzle.useController]({{ site_base_path }}sdk-reference/js/6/kuzzle/use-controller) method.

Reviewer: ```suggestion
Custom SDK controllers have to extend the `BaseController` class to be added with the [Kuzzle.useController]({{ site_base_path }}sdk-reference/js/6/kuzzle/use-controller) method.
```
